### ISSUE TITLE
Training/validation/test data division of ImageRecordIter

### ISSUE THREAD

**Issue by GaiYu0** [apache/incubator-mxnet#4698]
Hi! MXNet ImageRecordIter divides CIFAR 10 data into a training set of 50K samples and a test set of 10K samples. There are numerous papers that divide CIFAR 10 data into 45K training samples, 5K validation samples and 10K test samples. I would like to adopt an identical division in order to produce comparable results. Any solution? (I cannot abandon ImageRecordIter because I am using it for data argumentation) Thank you very much!

**Comment by phunterlau** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
